working/augmentation-libraries/feature-specification.md

```diff
@@ -1040,136 +765,28 @@ extension type A(int b) {
 other types in which case the extension type syntax will then be understood by
 users to be a primary constructor for the extension type.*
 
-The extension type's representation object is _not_ a variable, even though it
-looks and behaves much like one, and it cannot be augmented as such. It is a
-compile time error to have any augmenting declaration with the same name as the
-representation object.
+[primary constructors]:
+https://github.com/dart-lang/language/blob/main/working/2364%20-%20primary%20constructors/feature-specification.md
+
+The extension type's representation object is *not* a variable, even though it
+looks and behaves much like one, and it cannot be augmented as such.
+
+It's a **compile-time error** if:
 
-It is a compile time error if:
+*   An augmenting declaration has the same name as the representation object.
 
 *   An extension type augmentation contains a representation field clause.
 
-[primary constructors]:
-https://github.com/dart-lang/language/blob/main/working/2364%20-%20primary%20constructors/feature-specification.md
+### Augmenting with metadata annotations
 
-### Augmenting external members
-
-When augmenting an `external` member, it is assumed that a real implementation
-of that member has already been filled by some tool prior to any augmentations
-being applied. Thus, it is allowed to use `augmented` from augmenting members
-on external declarations, but it may throw a `NoSuchMethodError` error at
-runtime if no implementation was in fact provided.
-
-**NOTE**: Macros should _not_ be able to statically tell if an external body has
-been filled in by a compiler, because it could lead to a different result on
-different platforms or tools.
-
-**TODO: Should we add a syntax to let the augmentation dynamically detect
-whether there is an external implementation to call?**
-
-### Augmenting with metadata annotations and doc comments
```

Comment:
Why is this removed? This is fairly important

Reply:
It was redundant. The proposal says elsewhere that this works.